packages/client/components/AddTeamDialogRoot.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {Suspense} from 'react'
+import AddTeamDialog from './AddTeamDialog'
+import {renderLoader} from '../utils/relay/renderLoader'
+import useQueryLoaderNow from '../hooks/useQueryLoaderNow'
+import addTeamDialogQuery, {AddTeamDialogQuery} from '../__generated__/AddTeamDialogQuery.graphql'
+
+interface Props {
+  onClose: () => void
+  onAddTeam: (teamId: string) => void
+}
+
+const AddTeamDialogRoot = (props: Props) => {
+  const {onClose, onAddTeam} = props
+
+  const queryRef = useQueryLoaderNow<AddTeamDialogQuery>(addTeamDialogQuery, {}, 'network-only')
+
+  return (
+    <Suspense fallback={renderLoader()}>
+      {queryRef && (
+        <AddTeamDialog onAddTeam={onAddTeam} isOpen={true} onClose={onClose} queryRef={queryRef} />
```

Comment:
+1 normally one would mount the dialog and then only modify `isOpen` so we can get a transition.

Reply:
Transition works for me like this. Can you record a video?

I'll merge this PR in the mean time, and will fix it in next PRs if it is an issue